app/views/state_file/questions/az_senior_dependents/edit.html.erb

```diff
@@ -0,0 +1,46 @@
+<% content_for :page_title, "Dependents' 65+ Extra Questions" %>
+
+<% content_for :card do %>
+  <p><%= t('.title1') %></p>
+
+  <% if @form.errors.any? %>
+    <div class="notice--error">
+      <% @form.errors.full_messages.each do |msg| %>
+        <div><%= msg %></div>
+      <% end %>
+    </div>
+  <% end %>
+
+  <%= form_with model: @form, url: { action: :update }, method: :put, local: true, builder: VitaMinFormBuilder, class: 'form-card' do |f| %>
+    <%= f.fields_for :dependents do |ff| %>
+        <div class="highlight-box highlight-box--blue spacing-below-15">
+          <div class="spacing-below-5">
+            <%= ff.cfa_radio_set(
+                  :needed_assistance,
+                  label_text: t(".assistance_label", name: ff&.object&.first_name&.capitalize),
+                  collection: [
+                    { value: "yes", label: t(".affirmative") },
+                    { value: "no", label: t(".negative") },
+                  ]
+                ) %>
+          </div>
+        </div>
+        <div class="highlight-box highlight-box--blue spacing-below-15">
+          <div class="spacing-below-5">
+            <%= ff.cfa_radio_set(
+                  :passed_away,
+                  label_text: t(".passed_away_label", name: ff&.object&.first_name&.capitalize),
```

Comment:
I assume we will never have an empty object, so I don't believe the & accessor is needed to protect against an missing record.